src/main/java/com/buuz135/industrial/utils/BlockUtils.java

```diff
@@ -64,6 +67,14 @@
         return blocks;
     }
 
+    public static boolean isBlockTag(World world, BlockPos pos, ResourceLocation resource) {
+        Item item = world.getBlockState(pos).getBlock().asItem();
+        if (item == null) {
+            return false;
+        }
+        return item.getTags().contains(resource);
```

Comment:
This should just take a `Tag<Block>` instead of a `ResourceLocation` and then the method's body would look something like:
```java
public static boolean isBlockTag(World world, BlockPos pos, Tag<Block> tag) {
    return world.getBlockState(pos).isIn(tag);`
}
````